frontend/src/components/Map/styles.module.css

```diff
@@ -20,3 +20,33 @@
 .map {
   z-index: 1;
 }
+.legend {
+  background-color: rgba(153, 149, 149, 0.6);
+  border-radius: 3px;
+  bottom: 30px;
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.1);
+  font: 12px/20px "Helvetica Neue", Arial, Helvetica, sans-serif;
```

Comment:
There's way too many fonts in this project, and I'm pretty sure Helvetiva Neue isn't one of them. Can we use either Roboto or one of the other already existing fonts?

Reply:
oh, I stole this config from an example promising I would review then what was really necessary, and I completely forgot about it. Will change